app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -183,9 +191,11 @@ function initializePodcastPlayback() {
     Array.prototype.forEach.call(records, function (record) {
       var episodeSlug = record.getAttribute('data-episode');
       var podcastSlug = record.getAttribute('data-podcast');
-      record.onclick = function () {
+
+      var togglePodcastState = function (e) {
+        e.preventDefault();
```

Comment:
not a blocking issue, but I don't think we actually need `e.preventDefault()`, and it could safely be removed

Reply:
Yes, mainly added it to eliminate the default behaviour of Spacebar (page scroll) when this element was a `div`; but I guess it's already internally handled for a `button` element